test/testUtils/TestSuite.scala

```diff
@@ -47,7 +47,7 @@ trait TestSuite extends AnyWordSpec with Matchers with MockFactory with GuiceOne
   }
 
   implicit val actorSystem: ActorSystem = ActorSystem()
-  implicit val materializer: Materializer = ActorMaterializer()
+  implicit val materializer: SystemMaterializer = SystemMaterializer(actorSystem)
```

Comment:
Doe shis need to be here? I though we were just deleting it.

Reply:
I'm not sure really, I saw Abdullah do this and thought it was slightly better than removing it entirely but I don't think it particularly matters either way.